### ISSUE TITLE
Environmental covariates

### ISSUE THREAD

**Issue by ebuhle** [mdscheuerell/chumIPM#7]
@kalebentley and I had a conversation yesterday in which we discussed, among other things, prioritizing the tasks in this year's SOW. We agreed it makes sense to start with incorporating habitat / environmental covariates of various life-stage parameters (egg-to-smolt intrinsic productivity and capacity, SAR). I'd like to hear any input on what candidate variables to consider, data availability, etc.

Some initial thoughts:

- The easiest starting point, as we've [discussed previously](https://github.com/mdscheuerell/chumIPM/issues/3#issuecomment-609988259), would be to simply include an estimate of "total habitat" in each basin / population. In **salmonIPM** this can be used as an offset that scales the capacity parameter, rather than a covariate with an estimated slope. There is some nuance about what we mean by "habitat" -- initially we were thinking in terms of area used for spawning, but if we're now modeling [density dependence](https://github.com/mdscheuerell/chumIPM/issues/6) in egg-to-smolt survival (which I'm still working on b/c it hasn't gone as cleanly as I'd hoped) then I suppose technically it should be fry rearing habitat. Fortuitously, in the case of chum these are basically the same thing (someone correct me if I'm wrong). My understanding from @kalebentley is that these estimates should be available soon.

- Another relatively straightforward covariate, which we talked about in our meeting last August, is distance from tributary to mouth of the Columbia, hypothesized to be inversely related to SAR. It seems like this would be easy to estimate from GIS, perhaps alongside the pairwise tributary-to-tributary distances that we'll eventually need for attempting to estimate a straying matrix (a topic for another issue).

- We could always consider the usual oceanographic suspects like PDO, NPGO, etc. I'm not very familiar with the chum-specific literature on marine survival, so I'd be interested in hearing any thoughts on what specific oceanographic indicators would make sense.

- My impression is that freshwater habitat quality may be the most interesting and also the most difficult to quantify based on available data. We talked about simple categorizations, e.g. proportion of total habitat that is mainstem vs. side channel vs. artificial channel, etc. I'd imagine flow and temperature might be relevant as well, but I don't know if they're monitored across populations.

As we're thinking about this, keep in mind that any covariate that is identical across populations (e.g., oceanographic indicators) will have an effective sample size of 19, i.e. `N_year`. Likewise, any covariate that is population-specific but time-invariant (e.g., GIS-derived FW habitat classifications or mainstem Columbia travel distances) will have an effective sample size of `N_pop` (12, in the current configuration). So it behooves us to be parsimonious.

**Comment by Hillsont** [comment]
Eric,

Back before we started the IPM work with you we’d planned on getting all the ocean metric and trawl catch (they do catch and record juvenile chum in the trawls) data NOAA collects in their annual surveys off the OR and WA coasts to see if we could develop something similar to their “stoplight” chart and principle component analysis specific to LCR chum.  What NOAA publishes on their stoplight chart is geared towards Chinook.  Kris Homel started some collaboration efforts with NOAA staff on this topic when she was ODFWs LCR chum salmon recovery lead.  She has since left ODFW for a position with the Northwest Power and Planning Council (NPCC).  I sent here an e-mail today to see if she’s still interested in being involved with the chum IPM or not.  Either way, I’m sure she’ll pass along those NOAA contacts so we can get the data if interested or we can start at the top and ping Brian Burke about it.

Also, how much of the time under this year’s contract do you estimate incorporating habitat / environmental covariates of various life-stage parameters will take?  Maybe I’m missing something, or the order of steps that need to happen for things to move along, but I’d really like to see if we can start looking at why the upstream Columbia River hatchery programs (Duncan/Washougal and Lewis) are not performing anywhere near the level the lower river program (Grays).  The underwhelming performance of the Duncan and Lewis hatchery programs make them both targets for funding cuts and more importantly, I don’t want us to continue to mine the NOR populations for broodstock if the programs are going to fail at producing adult returns that at least replace themselves, or better yet, provide a lift to the mined population abundance.


Thanks

TH

From: ebuhle ***@***.***>
Sent: Wednesday, March 3, 2021 12:41 PM
To: mdscheuerell/chumIPM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [mdscheuerell/chumIPM] Environmental covariates (#7)


External Email

@kalebentley<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fkalebentley&data=04%7C01%7CTodd.Hillson%40dfw.wa.gov%7C18d15142239d4e2ff7fb08d8de84a0f8%7C11d0e217264e400a8ba057dcc127d72d%7C0%7C0%7C637504008525710724%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=4fwk9P5rHakfGknDpvtGXeCFuNC1Iiu1ydfkiawDGy0%3D&reserved=0> and I had a conversation yesterday in which we discussed, among other things, prioritizing the tasks in this year's SOW. We agreed it makes sense to start with incorporating habitat / environmental covariates of various life-stage parameters (egg-to-smolt intrinsic productivity and capacity, SAR). I'd like to hear any input on what candidate variables to consider, data availability, etc.

Some initial thoughts:

  *   The easiest starting point, as we've discussed previously<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmdscheuerell%2FchumIPM%2Fissues%2F3%23issuecomment-609988259&data=04%7C01%7CTodd.Hillson%40dfw.wa.gov%7C18d15142239d4e2ff7fb08d8de84a0f8%7C11d0e217264e400a8ba057dcc127d72d%7C0%7C0%7C637504008525720679%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=9cMWeuxFFdwxFHqeL6fV0nRNVIP%2Fdq6TcWf4N5ebKW8%3D&reserved=0>, would be to simply include an estimate of "total habitat" in each basin / population. In salmonIPM this can be used as an offset that scales the capacity parameter, rather than a covariate with an estimated slope. There is some nuance about what we mean by "habitat" -- initially we were thinking in terms of area used for spawning, but if we're now modeling density dependence<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmdscheuerell%2FchumIPM%2Fissues%2F6&data=04%7C01%7CTodd.Hillson%40dfw.wa.gov%7C18d15142239d4e2ff7fb08d8de84a0f8%7C11d0e217264e400a8ba057dcc127d72d%7C0%7C0%7C637504008525720679%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=8I4yVLH0Oq2aBTOVX8GB%2BrYSQ5lKrwyR4%2Bv59uFZ46o%3D&reserved=0> in egg-to-smolt survival (which I'm still working on b/c it hasn't gone as cleanly as I'd hoped) then I suppose technically it should be fry rearing habitat. Fortuitously, in the case of chum these are basically the same thing (someone correct me if I'm wrong). My understanding from @kalebentley<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fkalebentley&data=04%7C01%7CTodd.Hillson%40dfw.wa.gov%7C18d15142239d4e2ff7fb08d8de84a0f8%7C11d0e217264e400a8ba057dcc127d72d%7C0%7C0%7C637504008525730628%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=B%2Bl8vJbLmoHhgSmQb3%2FkRf%2FhntHdEkBeL4gOS%2Bpru04%3D&reserved=0> is that these estimates should be available soon.
  *   Another relatively straightforward covariate, which we talked about in our meeting last August, is distance from tributary to mouth of the Columbia, hypothesized to be inversely related to SAR. It seems like this would be easy to estimate from GIS, perhaps alongside the pairwise tributary-to-tributary distances that we'll eventually need for attempting to estimate a straying matrix (a topic for another issue).
  *   We could always consider the usual oceanographic suspects like PDO, NPGO, etc. I'm not very familiar with the chum-specific literature on marine survival, so I'd be interested in hearing any thoughts on what specific oceanographic indicators would make sense.
  *   My impression is that freshwater habitat quality may be the most interesting and also the most difficult to quantify based on available data. We talked about simple categorizations, e.g. proportion of total habitat that is mainstem vs. side channel vs. artificial channel, etc. I'd imagine flow and temperature might be relevant as well, but I don't know if they're monitored across populations.

As we're thinking about this, keep in mind that any covariate that is identical across populations (e.g., oceanographic indicators) will have an effective sample size of 19, i.e. N_year. Likewise, any covariate that is population-specific but time-invariant (e.g., GIS-derived FW habitat classifications or mainstem Columbia travel distances) will have an effective sample size of N_pop (12, in the current configuration). So it behooves us to be parsimonious.

**Comment by tbuehrens** [comment]
Hey @ebuhle sorry for the slow reply!!! Couple thoughts:
1) yes, I'd use total habitat as an offset (not covariate)..specifically total area (sq m) of previously occupied/used spawning habitat. Later we can think about including "potential habitat"
2) yes, I'd use migration distance to river mouth...I probably would not do this in logit space but something like fry to adult = ms[y] * rs^distance, where rs is the river survival per mile or per 100 miles (so rs isn't too close to 1) which is constrained to 0-1. I haven't played around with the logit transformation so it may be that there is an equivalent way to do this in logit space.
3) spawning habitat type would be a good global categorical variable.
4) in terms of annual habitat covariates, I think flow variables would be good: a) log(max flow recurrence interval), b) max(z-scored stage height...would have to pick "representative gauges", which is fraught. I'd also expect an interaction between flow and habitat type (big flow = bigger problem in mainstem vs off channel).
5) yes we can throw the kitchen sink of ocean covariates...or start with some PC axes of them

**Comment by kalebentley** [comment]
As I mentioned the other day, with the help of several other people including Steve VanderPloeg (SVP), @Hillsont and @BradGarnerWDFW, I was finally able to pull together various data sources to generate estimates of total chum spawning habitat for each population that we currently have in our IPM (data -> "Data_Habitat_Spawning_Linear_2021-04-15.csv")  

An estimate of spawning habitat has been generated for each brood year though, in general, the estimate is the same across all years.  A few notable exceptions are (1) Duncan Channels: corresponding to the three periods and the associated habitat modification outlined [here ](https://github.com/mdscheuerell/chumIPM/issues/5#issue-682273616), (2) Hamilton Channel/Creek: per the details @Hillsont outlined [here](https://github.com/mdscheuerell/chumIPM/issues/6#issuecomment-808457063) though upon closer examination of the final habitat data file posted - it looks like the summary isn't quite right and may need to be modified slightly (I work with @Hillsont to get this fixed ASAP), and (3) Grays_CJ: there was some habitat work completed in the last couple of years that increased the habitat in CJ but again it doesn't look like the final data file captured this (UGH! -- I'm sorry -- I'll get this fixed ASAP).  

As we see fit, we can obviously update the current habitat file with year-to-year changes in habitat (assuming this can be done in an objective and consistent manner).

Currently, the units of spawning habitat are in linear miles.  This is also something we can revisit at a later time if warranted.  However, based on conservations with @Hillsont and others, the "width" of habitat used by spawners site-to-site is generally pretty similar among populations/sites even at mainstem spawning sites.

The estimates of spawning habitat were generated in GIS using GPSed chum redds that were collected during spawning grounds.  SVP and I went through several iterations of this process and so I will have to double-check with him regarding the exact make-up of the redd dataset (i.e., number of years by population) but, in short, GPSed redds across all years were compiled and linear distances were calculated by adding the distance from the upper-most redd down to the lowest-most redd for each survey segment and then summed across all segments for a given population.  

Below is an example of how this was done in the Grays River watershed for the three associated "sub" populations.  In the map, the green dots are the individual GPSed redds across all years.  I've added colored circles for illustrative purposes showing the upper and lower most redd for Grays_WF (yellow), Grays_CJ (blue), and Grays_MS (purple).  
![image](https://user-images.githubusercontent.com/40397762/115476076-e79fc880-a1f5-11eb-9254-43e929f71492.png)

Again, I'll have to double-check with SVP and @Hillsont, but I believe the GPS redd data only go back a few years (maybe 2017?).  Therefore, the legend in the map above that denotes years (2000-2019) is a tiny bit misleading.  Technically, redd data have been conducted since 2000 - redds have been tallied by survey section (notice the different colored segments - gray, blue, yellow, orange, red, and dark red -  in the map above) -- see my explanation in the second to last paragraph below regarding an alternative spawning habitat metric that we could explore using later.

One quick aside regarding the Grays habitat estimates - you may notice that there's a tributary (Hull Creek) that comes into the Gray_MS below the juvenile monitoring weir.  This area was not included in our estimate of habitat because spawner surveys from these sections are NOT included in the estimates of adult abundance.  Certainly a conversation for another day...

A similar process was completed for mainstem populations except here we used a combination of both redd data and "professional" opinion.  In reality, there wasn't a ton of professional opinion needed but it's worth noting that the mainstem redd surveys (and associated GPS redd data) are not conducted with abundance estimates in mind and thus do not include all known spawning areas.  

Lastly, it's worth noting that SVP calculated a second metric of spawning habitat actually using all redd data (e.g., dating back to 2000 in the Grays Basin) by tallying the length of segments where at least one redd has been detected across all years.  The estimates of spawning habitat between the two metrics (upper/lower GPS redd vs. segment with a redd) are (not surprisingly) a bit different but there is generally a linear relationship between the two metrics across watersheds.  As we add more populations to the IPM, we'll probably need to assess how we calculate spawning habitat because there won't necessarily be consistent datasets available (e.g., redd data don't exist for all populations).

At some point soon, I'll try to better summarize how these habitat metrics were generated.  In the meantime, let me know what questions/concerns you have and what, if any, modifications we want to make to these estimates of spawning habitat moving forward.

** I apologize for any grammatical errors in the post above -- I had to hastefully review.

**Comment by Hillsont** [comment]
First off I want to say I'm not intentionally trying to make this more complicated, I just want to get all the cards on the table and and let those with bigger brains than mine make decisions.  

Chum spawner distribution data based on redds should exist from 2001-2003 I believe.  WDFW was working with PNNL on a Grays River Watershed Geomorphic Analysis (https://www.osti.gov/biblio/909257), I wasn't directly involved with this but I do know some details.  Then when WDFW began collecting survey data using iPads (2017ish I think) we started to collect GIS data again on redds, between those time periods redd counts were sectioned out by one mile reaches in tribs.  Chum tend to mass spawn spawn so individual redd data was not only not collected but not very useful/accurate for generating spawner estimates like it could be for Chinook and coho.

The habitat work (spawning channel creation) in CJ was done in the early fall of 2017 by the Lower Columbia Fish Enhancement Group.

Lastly, prior to the evulsion of the mainstem Grays River through the Gorley Dike in the winter of 1998-99, CJ proper dumped into the MS Grays River.  What we now call the "CJ spawning area" (CJ2 in TWS) is actually an old MS Grays overflow channel that ends in a head cut that gets closer to the MS Grays each year since the MS decided to reconnect to the overflow channel back in 2013.  There's been a couple of habitat projects to slow this "capture" down but it continues during high flow events and CJ spawning area gets a bit bigger every year as a result.  CJ proper has been disconnected (flow went underground due to culvert failure and landslide) since the late 1990s until fall of 2015 when it re-connected with the CJ spawning area via overland flows (CJ3 in TWS).  Aside from the fall of 2016, I don't believe CJ proper (CJ3 & CJ4) has been used for spawning by chum.
